code/modules/paperwork/pen.dm

```diff
@@ -139,10 +139,32 @@ var/paperwork_library
 /datum/writing_style/pen/New()
 	addReplacement(REG_BBTAG("*"), "<li>")
 	addReplacement(REG_BBTAG("hr"), "<HR>")
-	addReplacement(REG_BBTAG("small"), "<font size = \"1\">")
-	addReplacement(REG_BBTAG("/small"), "</font>")
+	addReplacement(REG_BBTAG("small"), "<span style=\"font-size:15px\">")
+	addReplacement(REG_BBTAG("/small"), "</span>")
+	addReplacement(REG_BBTAG("tiny"), "<span style=\"font-size:10px\">")
+	addReplacement(REG_BBTAG("/tiny"), "</span>")
 	addReplacement(REG_BBTAG("list"), "<ul>")
 	addReplacement(REG_BBTAG("/list"), "</ul>")
+	addReplacement(REG_BBTAG("agency"),  	"<span style=\"font-family:Agency FB\">")
+	addReplacement(REG_BBTAG("/agency"), 	"</span>")
+	addReplacement(REG_BBTAG("algerian"),  	"<span style=\"font-family:Algerian\">")
+	addReplacement(REG_BBTAG("/algerian"), 	"</span>")
+	addReplacement(REG_BBTAG("arial"),  	"<span style=\"font-family:Arial\">")
+	addReplacement(REG_BBTAG("/arial"), 	"</span>")
+	addReplacement(REG_BBTAG("arialb"),  	"<span style=\"font-family:Arial Black\">")
+	addReplacement(REG_BBTAG("/arialb"), 	"</span>")
```

Comment:
You know what annoys me about this?
It isn't true.
The equivalent of a font break isn't a span end, it's that specific span end.
This makes 
`[arialb][small]This is small arial bold[/arialb] but this isn't small[/small]`
That doesn't make any sense.